_assets/js/templates/ellipsisTemplate.js

```diff
@@ -0,0 +1,8 @@
+const ellipsis = () => {
+  return `<li
+    class="usa-pagination__item usa-pagination__overflow"
+    role="presentation"
+  >
+    <span> … </span>
+  </li>`;
+};
```

Comment:
I wonder if we should put all these templates in a pagination folder?  What do you think?  Would they be used in any other context?

Reply:
I like this idea. I can definitely do that. Components that are reused across pages can live in the top level templates dir, and then anything that is page/function specific can be drilled down into a subdir.

Reply:
This is complete. See new commit.